chaudloader/src/hooks/stage1.rs

```diff
@@ -189,3 +206,49 @@ pub unsafe fn install() -> Result<(), anyhow::Error> {
 
     Ok(())
 }
+
+/// Install optional on_game_laod hook into the process.
+pub unsafe fn install_on_game_load() -> Result<(), anyhow::Error> {
+    unsafe {
+        let module =  windows_libloader::ModuleHandle::get(&std::env::current_exe()?.to_string_lossy())?
+                .get_base_address() as *const u8;
+
+        let section = object::read::pe::PeFile64::parse(
+            std::slice::from_raw_parts(
+                module, 0x1000,
+            ))?.section_table().section(1)?;
+        
+        let (start, size) = section.pe_address_range();
```

Comment:
Instead of duplicating this code from `stage0::process_game_sections`, pass `game_env` into this function from `stage0::init`, from which you can pull `game_env.sections.text`.